static/third/bootstrap/css/bootstrap.css

```diff
@@ -316,14 +308,6 @@ pre {
   -moz-border-radius: 4px;
   border-radius: 4px;
 }
-pre code {
-  padding: 0;
```

Comment:
Our `pre code` rule above doesn't have `padding: 0`; is that a change?

Reply:
I missed this since there are no actual instances of our `pre code` rule above being applied in the web-app. I also need to move the background-color rule as well.